site/documentation/content/api/event/index.md

```diff
@@ -134,3 +130,21 @@ The example below might be used by the MQTT adapter to indicate that a connectio
   }
 }
 ~~~
+
+### Device Provisioning Notification
+
+Device registries may send this event to indicate that a new device has been provisioned. This may be used as a hook for 
+northbound applications if gateway-based auto-provisioning is enabled and further application specific logic should be
+implemented upon a newly provisioned device.
+
+An AMQP message containing this type of event does not have any payload, so the body of the message MUST be empty.
+
+The relevant properties are listed again in the following table:
+
+| Name                       | Mandatory        | Location                 | Type      | Description |
+| :------------------------- | :--------------: | :----------------------- | :-------- | :---------- |
+| *content-type*             | yes              | *properties*             | *symbol*  | MUST be set to *application/vnd.eclipse-device-provisioning-notification* |
+| *hono_registration_status* | no               | *application-properties* | *string*  | If set to `NEW` this indicates that a device was auto-provisioned in the course of sending telemetry data via gateway. |
+| *tenant_id*                | no               | *application-properties* | *string*  | The tenant id denoting the tenant for which the device was auto-provisioned. |
+| *device_id*                | no               | *application-properties* | *string*  | The device id denoting of the auto-provisioned device. |
```

Comment:
FMPOV this is mandatory because without it a consumer does not know which device we are talking about, does it?
We should not mention *auto-provisioned* here as the event is also applicable for manual provisioning cases ...